cirq-core/cirq/protocols/circuit_diagram_info_protocol.py

```diff
@@ -188,7 +188,7 @@ class CircuitDiagramInfoArgs:
             properly handle unicode characters.
         precision: The number of digits after the decimal to show for numbers in
             the text diagram. None means use full precision.
-        qubit_map: The map from qubits to diagram positions.
+        label_map: The map from label entities to diagram positions.
```

Comment:
I think it's worth defining "label entity" here (IIUC, it's a `Qid` or a `MeasurementKey`).